language/tools/test_generation/src/control_flow_graph.rs

```diff
@@ -286,7 +286,7 @@ impl CFG {
                 block_queue.push_back(child_ids[1]);
             } else if child_ids.len() == 1 {
                 block_queue.push_back(child_ids[0]);
-            } else if child_ids.is_empty() {
+            } else if !child_ids.is_empty() {
```

Comment:
Whoops. How did that get past testing?

Reply:
The tests check the instructions' preconditions and effects, there isn't a test that constructs an entire CFG unfortunately.